doc/api/net.md

```diff
@@ -386,7 +386,7 @@ Creates a new socket object.
     for details. Defaults to `false`.
   * `readable` {boolean} Allow reads on the socket when a `fd` is passed,
     otherwise ignored. Defaults to `false`.
-  * `writable` {boolean} Allow reads on the socket when a `fd` is passed,
+  * `writable` {boolean} Allow writes on the socket when a `fd` is passed,
```

Comment:
I suppose it depends on whether you read it as "an ef dee" or "a file descriptor."